### PR TITLE
bump polars v0.5.0 + news

### PR DESCRIPTION
**Pull request by sorhawell**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: polars
 Title: Polars ported to R
-Version: 0.4.8.9000
+Version: 0.5.0.9000
```
Comment (eitsupi) [inline DESCRIPTION:3]:
```suggestion
Version: 0.5.0
```

File: NEWS.md [hunk 2]
```diff
@@ -1,9 +1,22 @@
...
-## BREAKING CHANGE
+# polars v0.5.0
 
+## BREAKING CHANGE
 - The package name was changed from `rpolars` to `polars`. (#84)
 
+## What's changed
+- Several new methods for DataFrame, LazyFrame & GroupBy translated (#103, #105 @vincentarelbundock)
+- Doc fixes (#102, #109  @etiennebacher)
+- Experimental opt-in auto completion by (#96 @sorhawell)
```
Comment (eitsupi) [inline NEWS.md:11]:
```suggestion
- Experimental opt-in auto completion (#96 @sorhawell)
```

**Review by eitsupi — APPROVED** [review]
I recommend that the version number be changed with the `usethis::use_version()` function.

**Comment by eitsupi** [comment]
@sorhawell Are you planning to release 0.5.0?

**Comment by sorhawell** [comment]
@eitsupi  yes I will release tonight.

**Pull request merged**